frontend/src/components/Admin/TeamEvent/TeamEventDashboard.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect } from 'react';
+import { Table, Header, Loader } from 'semantic-ui-react';
+import { useMembers } from '../../Common/FirestoreDataProvider';
+import { TeamEventsAPI } from '../../../API/TeamEventsAPI';
+import { REQUIRED_LEAD_TEC_CREDITS, REQUIRED_MEMBER_TEC_CREDITS } from '../../../consts';
+import styles from './TeamEventDashboard.module.css';
+
+const calculateMemberCreditsForEvent = (member: IdolMember, event: TeamEvent): number =>
+  event.attendees.reduce((val: number, attendee) => {
+    if (attendee.member.email !== member.email) {
+      return val;
+    }
+    if (event.hasHours && attendee.hoursAttended)
+      return val + Number(event.numCredits) * attendee.hoursAttended;
+    return val + Number(event.numCredits);
+  }, 0);
+
+const TeamEventDashboard: React.FC = () => {
+  const [teamEvents, setTeamEvents] = useState<TeamEvent[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+
+  const allMembers = useMembers();
+
+  useEffect(() => {
+    TeamEventsAPI.getAllTeamEvents().then((events) => {
+      setTeamEvents(events);
+      setIsLoading(false);
+    });
+  }, []);
+
+  if (isLoading) return <Loader active>Fetching team event data...</Loader>;
+
+  return (
+    <div className={styles.dashboardContainer}>
+      <Header as="h1" textAlign="center">
+        Team Event Dashboard
+      </Header>
+      <div className={styles.tableContainer}>
+        <Table celled selectable striped classname={styles.dashboardTable}>
+          <Table.Header>
+            <Table.HeaderCell className={styles.nameCell}>Name</Table.HeaderCell>
+            <Table.HeaderCell>Total</Table.HeaderCell>
```

Comment:
I would add community total too, but figure out a way to hide it for now.

Reply:
sure, didn't notice that in the ticket